addons/website_slides/static/src/xml/activity.xml

```diff
@@ -1,22 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
-    <t t-extend="mail.activity_items">
-        <t t-jquery=".o_thread_message .o_thread_message_core .o_thread_message_tools" t-operation="replace">
-            <t t-if="activity.activity_type_id[1] == 'Access Request'">
-                <t t-if="activity.user_id[0] === uid">
-                    <a role="button" class="btn btn-link btn-success text-muted text-success o_activity_link o_activity_action_grant_access" t-att-data-partner-id="activity.request_partner_id[0]">
+    <t t-name="website_slides.Activity" t-inherit="mail.Activity" t-inherit-mode="extension">
+        <xpath expr="//div[@name='tools']" position="replace">
+            <t t-if="activity.type and activity.type.displayName === 'Access Request'">
```

Comment:
Not your fault but this condition is crap/not working.
Testing against a string is unreliable and does not work with any other language than English.

Also see approval in enterprise for how we handled it.

Reply:
Yes! But in the `approvals` module, we have a new structure like (components, models....) and here we need to fix it in an existing file. For approvals, there is one new one2many field added which is the inverse of 'activity' and handled it by that field.

Could I manage all these in the existing file `website_slides/static/src/js/activity.js`?

Reply:
Yes, you can put everything on the existing file: new component/override, new model/override, ...

We will split correctly in master.

Reply:
@seb-odoo changes are done.